rules/signing.bzl

```diff
@@ -948,3 +948,46 @@ keyset = rule(
         ),
     },
 )
+
+def _signature_test_impl(ctx):
+    _script = ctx.actions.declare_file("{}.bash".format(ctx.label.name))
```

Comment:
I'd name this just `script` (without the underscore).  This `script` is the output (e.g. the template-expanded form of `_script`).